pkg/llm-d-inference-sim/simulator.go

```diff
@@ -511,11 +511,14 @@ func (s *VllmSimulator) HandleError(_ *fasthttp.RequestCtx, err error) {
 	s.logger.Error(err, "VLLM server error")
 }
 
-// createCompletionResponse creates response for completion request, supports both completion requests types (text and chat) as defined by isChatCompletion
-// respText - content to be sent in the response
+// createCompletionResponse creates response for completion request, supports both completion requests types (text and chat)
```

Comment:
Change text to:

createCompletionResponse creates the response for completion requests, supports both completion request types (text and chat)